tests/python/frontend/onnx/test_forward.py

```diff
@@ -1070,6 +1073,47 @@ def test_LogSoftmax():
                               'LogSoftmax',
                               {'axis': 1})
 
+def check_torch_conversion(model, input_size):
+    dummy_input = torch.randn(*input_size)
+    file_name = '{}.onnx'.format(model.__name__)
+    # Set verbose=True for more output
+    torch.onnx.export(model(), dummy_input, file_name, export_params=True, verbose=False)
+    onnx_model = onnx.load(file_name)
+    shapes = { '0' : input_size }
+    expr, params = relay.frontend.from_onnx(onnx_model, shape=shapes)
+
+def test_resnet():
+    check_torch_conversion(torchvision.models.resnet18, (1,3,224,224))
+    # check_torch_conversion(torchvision.models.resnet101, (1,3,224,224))
+
+# def test_alexnet():
```

Comment:
What's the reason for these tests being commented out? Can we add a TODO if it's pending on some feature that isn't implemented yet?